packages/hackmd/dist/agent.js

```diff
@@ -0,0 +1 @@
,l=p.parentOrigin,f=l===void 0?"*":l,O=p.methods,c=O===void 0?{}:O,h=p.timeout,D=p.debug;const S=(0,m.default)(D);if(window===window.top){const w=new Error("connectToParent() must be called within an iframe");throw w.code=n.ERR_NOT_IN_IFRAME,w}const T=(0,s.default)(),g=T.destroy,R=T.onDestroy,v=window,C=v.parent;return{promise:new Promise((w,P)=>{let M;h!==void 0&&(M=setTimeout(()=>{const o=new Error(`Connection to parent timed out after ${h}ms`);o.code=n.ERR_CONNECTION_TIMEOUT,P(o),g()},h));const I=o=>{try{clearTimeout()}catch{return}if(o.source!==C||o.data.penpal!==t.HANDSHAKE_REPLY)return;if(f!=="*"&&f!==o.origin){S(`Child received handshake reply from origin ${o.origin} which did not match expected origin ${f}`);return}S("Child: Received handshake reply"),v.removeEventListener(t.MESSAGE,I);const y={localName:"Child",local:v,remote:C,originForSending:o.origin==="null"?"*":o.origin,originForReceiving:o.origin},x={},K=(0,i.default)(y,c,S);R(K);const b=(0,u.default)(x,y,o.data.methodNames,g,S);R(b),clearTimeout(M),w(x)};v.addEventListener(t.MESSAGE,I),R(()=>{v.removeEventListener(t.MESSAGE,I);const o=new Error("Connection destroyed");o.code=n.ERR_CONNECTION_DESTROYED,P(o)}),S("Child: Sending handshake"),C.postMessage({penpal:t.HANDSHAKE,methodNames:Object.keys(c)},f)}),destroy:g}};e.default=_,r.exports=e.default})(F,F.exports);const ae=j(F.exports);function U(r,e,t,n){var s,i=!1,u=0;function m(){s&&clearTimeout(s)}function d(){m(),i=!0}typeof e!="boolean"&&(n=t,t=e,e=void 0);function _(){for(var E=arguments.length,p=new Array(E),l=0;l<E;l++)p[l]=arguments[l];var f=this,O=Date.now()-u;if(i)return;function c(){u=Date.now(),t.apply(f,p)}function h(){s=void 0}n&&!s&&c(),m(),n===void 0&&O>r?c():e!==!0&&(s=setTimeout(n?h:c,n===void 0?r-O:r))}return _.cancel=d,_}function de(r,e,t){return t===void 0?U(r,e,!1):U(r,t,e!==!1)}const se=!1,ce="<%= origin %>";function le(){return window.editor.doc.getValue()}function z(r){window.editor.doc.setValue(r)}function ue(r){if(window.cmClient!=null){z(r);return}const e=setInterval(()=>{window.cmClient!=null&&(clearInterval(e),z(r))},250)}function Ee(r){window.growi.notifyBodyChanges(r)}const fe=de(800,Ee);function ge(r){window.growi.saveWithShortcut(r)}function Re(){const r=window.CodeMirror,e=window.editor;r==null||e==null||(e.on("change",(t,n)=>{n.origin!=="ignoreHistory"&&fe(t.doc.getValue())}),r.commands.save=function(t){ge(t.doc.getValue())},delete e.options.extraKeys["Cmd-S"],delete e.options.extraKeys["Ctrl-S"])}function _e(){ae({parentOrigin:ce,methods:{getValue(){return le()},setValue(e){z(e)},setValueOnInit(e){ue(e)}},debug:se}).promise.then(e=>{window.growi=e}).catch(e=>{console.log(e)})}(function(){if(window===window.parent){console.log("[GROWI] Loading agent for HackMD is not processed because currently not in iframe");return}console.log("[HackMD] Loading GROWI agent for HackMD..."),window.addEventListener("load",()=>{Re()}),_e(),console.log("[HackMD] GROWI agent for HackMD has successfully loaded.")})();
```

Comment:
I don't know why these files appeared.

Could you pull the parent branch `imprv/gw7848-imprv-bookmarks-content-on-user-page` to bring it up to date `feat/gw7910-add-bookmark-from-sub-navigation`?